bindings/python/CMakeLists.txt

```diff
@@ -84,19 +32,18 @@ target_compile_options(
   manifold3d
   PRIVATE ${MANIFOLD_FLAGS} -DMODULE_NAME=manifold3d
 )
-target_compile_features(manifold3d PUBLIC cxx_std_17)
 set_target_properties(manifold3d PROPERTIES OUTPUT_NAME "manifold3d")
 
 message(Python_EXECUTABLE = ${Python_EXECUTABLE})
 # ideally we should generate a dependency file from python...
 set(
   DOCSTRING_DEPS
-  ${CMAKE_SOURCE_DIR}/src/manifold.cpp
-  ${CMAKE_SOURCE_DIR}/src/constructors.cpp
-  ${CMAKE_SOURCE_DIR}/src/sort.cpp
-  ${CMAKE_SOURCE_DIR}/src/cross_section/cross_section.cpp
-  ${CMAKE_SOURCE_DIR}/src/polygon.cpp
-  ${CMAKE_SOURCE_DIR}/include/manifold/common.h
+  ${PROJECT_SOURCE_DIR}/src/manifold.cpp
+  ${PROJECT_SOURCE_DIR}/src/constructors.cpp
+  ${PROJECT_SOURCE_DIR}/src/sort.cpp
+  ${PROJECT_SOURCE_DIR}/src/cross_section/cross_section.cpp
+  ${PROJECT_SOURCE_DIR}/src/polygon.cpp
+  ${PROJECT_SOURCE_DIR}/include/manifold/common.h
```

Comment:
Ah, good catch.  Using CMAKE_SOURCE_DIR is a habit I have left from the old days, looks like PROJECT_SOURCE_DIR is probably what one would want most of the time.